drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -146,5 +146,10 @@ reportAvailableLabel=Available decisions:
 reportExecutedLabel=Tested decisions:
 reportCoverageLabel=Percentage of decisions tested:
 numberOfTimesDecisionEvaluated=A number of times each decision has been evaluated:
-
-
+#
+errorReason=Error reason
+keep=Keep
+close=Close
+apply=Apply
+errorPopoverMessageFailedWithError=The expected value is ''{0}'' but the actual one is ''{1}''.
+errorPopoverMessageFailedWithException=Error with this value: {0}
```

Comment:
Not sure if this comes form UX mockups, but I would expect something more like -  "Exception with detailed message: {0}"

Reply:
In my knowledge, this case is not handled in UX mockups.
Fine for me to put yours.
DONE!